docs/nima/pico/pico_intro.adoc

```diff
@@ -101,16 +175,26 @@ The Pico API provides types that are generally useful at compile time to assign
 
 == Configuration
 
-Pico Config builder
```

Comment:
Pico Configuration should be instead referred to as "Pico Config-Driven Services"